onnx/common/path.h

```diff
@@ -8,6 +8,9 @@
 
 #include <string>
 #ifdef _WIN32
+#ifndef NOMINMAX
```

Comment:
Why is it needed? Maybe add a comment in the code would help.

Reply:
I added a comment.